pkg/ui/workspaces/cluster-ui/webpack.config.js

```diff
@@ -143,7 +144,9 @@ module.exports = {
       amd: "react-dom",
       root: "ReactDom",
     },
-    "react-router-dom": "react-router-dom",
+    // TODO (koorosh): commented to avoid bazel build failure.
+    // should be fixed to consume react-router-dom as peer dependency
+    // "react-router-dom": "react-router-dom",
```

Comment:
Same question -- can this safely be commented out without breaking the `make` build?

Reply:
Done, I added condition in webpack configuration to exclude this change for bazel build only. Additional `is_bazel_build` env variable is passed to coordinate this flow.